bootloader/src/pyi_utils.c

```diff
@@ -740,6 +740,19 @@ pyi_utils_set_environment(const ARCHIVE_STATUS *status)
     return rc;
 }
 
+int
+set_systemd_env(pid_t pid)
+{
+    #define ULONG_STRING_SIZE (sizeof (unsigned long) * CHAR_BIT / 3 + 2)
```

Comment:
emm, I don't know what the pid_str size I should set, so this line roughly calculates the max decimal string length...Any better idea?